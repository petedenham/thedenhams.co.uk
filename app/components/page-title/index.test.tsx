import { render, screen } from "@testing-library/react";
import PageTitle from ".";

describe("PageTitle", () => {
  test("matches snapshot", () => {
    const { container } = render(<PageTitle>Title</PageTitle>);

    expect(container).toMatchSnapshot();
  });
});

