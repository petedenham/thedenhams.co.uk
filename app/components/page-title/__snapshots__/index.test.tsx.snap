// Vitest Snapshot v1

exports[`PageTitle > matches snapshot 1`] = `
<div>
  <h1
    class="text-xl mb-7 mt-2 font-bold text-center md:text-2xl md:mb-10 md:mt-5"
  >
    Title
  </h1>
</div>
`;
